src/CoinLockerSystem.java

```diff
@@ -63,25 +65,15 @@ private void rentLocker() {
         System.out.println("Your locker number is " + locker.getId());
         System.out.println("Price is " + locker.getPrice());
         getMoney(locker.getPrice());
-    }
-
-    private void getMoney(int price) {
-        double paid = 0;
-        Scanner scanner = new Scanner(System.in);
-        do {
-            System.out.println("Insert " + (price - paid) + " dollars.");
-            paid += scanner.nextDouble();
-        } while (paid < price);
-        if (paid > price) {
-            System.out.println("Here is your change " + (paid - price) + " dollars");
-        }
+        savePassword(locker);
+        System.out.println("Locker number " + locker.getId() + " is opened!");
     }
 
     private LockerSize getLockerSize() {
         System.out.println("What size do you want?");
         printSize();
-        Scanner scanner = new Scanner(System.in);
         int input = scanner.nextInt();
+        scanner.nextLine();
```

Comment:
lets wrap this logic into GetUserInput interface

Reply:
I'll do it later